sdks/csharp/authentication.cs

```diff
@@ -29,8 +29,28 @@ internal void RunSample()
                 Environment.Exit(0);
             }
         }
-        // </DefaultAzureCredential_full>
     }
+    // </DefaultAzureCredential_full>
+
+    // ------------------ DefaultAzureCredential with options  ------------------ 
+    // <DefaultAzureCredential_options>
+    public class DefaultAzureCredentialOptionsSample
+    {
+        // The URL of your instance, starting with the protocol (https://)
+        private const string adtInstanceUrl = "https://<your-Azure-Digital-Twins-instance-URL>";
+
+        private static DefaultAzureCredentialOptions credOpts = new DefaultAzureCredentialOptions()
+        {
+            ExcludeSharedTokenCacheCredential = true,
+            ExcludeVisualStudioCodeCredential = true,
+            TenantId = "a000a000-0aaa-0000-00a0-aa000aa0aa0a"
```

Comment:
Should we have an angled bracket placeholder here as well to indicate that this should be replaced? E.g.: `<Azure Active Directory Tenant Id>`